docs/manual-EN/server-administration/storage-balance.md

```diff
@@ -0,0 +1,219 @@
+-  The second column shows the state of the task, there are four states at present
+  - Succeeded: Operation succeed
+  - Failed: Operation failed
+  - In progress: Operating
+  - Invalid: Invalid task
+
+The last row is the summary of the tasks, some partitions are yet to be migrated.
+### Step 4 Migration confirmation
+In most cases, data migration is a long process. Fortunately, the existing services won't be affected. Once migration is done, the progress will show 100%. Rerun `balance data` to fix failed tasks. If they can't be fixed after several attempts, please contact us at [GitHub](https://github.com/vesoft-inc/nebula/issues).
+
+Check partition distribution using command `SHOW HOSTS` when balance completed.
+
+```
+(user@127.0.0.1) [(none)]> SHOW HOSTS
+
+================================================================================================
+| Ip            | Port  | Status | Leader count | Leader distribution | Partition distribution |
+================================================================================================
+| 192.168.8.210 | 34600 | online | 3            | test: 3             | test: 37               |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34900 | online | 0            | test: 0             | test: 38               |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 35940 | online | 0            | test: 0             | test: 37               |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34920 | online | 0            | test: 0             | test: 38               |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 44920 | online | 0            | test: 0             | test: 38               |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34700 | online | 35           | test: 35            | test: 37               |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34500 | online | 24           | test: 24            | test: 37               |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34800 | online | 38           | test: 38            | test: 38               |
+------------------------------------------------------------------------------------------------
+Got 8 rows (Time spent: 5074/6488 us)
+```
+
+Now partitions and data are evenly distributed on the machines.
+
+## Balance leader
+Balance data only balances partition but the leader distribution remains unbalanced, which means old services are overloaded while the new ones are not fully used. Redistribute RAFT leader using the command `balance leader`.
```

Comment:
Command `balance data` only migrate partitions. But the ...

old hosts are overloaded, while